README.rst

```diff
@@ -125,15 +122,14 @@ Examples Directory structure
 If you are looking for UI,checkout `Modbus Simulator <https://github.com/riptideio/modbus-simulator>`_ or
 `Modbus Cli <https://github.com/dhoomakethu/modbus_sim_cli>`_
 
-Also, if you have questions, please ask them on the mailing list
-so that others can benefit from the results and so that I can
-trace them. I get a lot of email and sometimes these requests
+Also, if you have questions, please create an issue so that others can benefit from the results and so that we can
```

Comment:
Better:

Also, if you have a question, please [create an post in discussions q&a topic](https://github.com/riptideio/pymodbus/discussions/new?category=q-a), so that others can benefit from the results [...].

If you think, that something in the code is broken/not running well, please [open an issue](https://github.com/riptideio/pymodbus/issues/new), read the Template-text first and then post your issue with your setup informations.

Reply:
Done.